<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="RemoveStopwords">
     <property name="text">
      <string>Remove Stopwords</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="StaticLang">
     <property name="toolTip">
      <string>Set if the languages are static.</string>
     </property>
     <property name="whatsThis">
      <string>When the languages are static and when the content of a document is changed, the system supposes that the language of this document has not changed.&lt;br&gt;
When the languages are not static, the language of a document is always determinewhen the document is analysed.</string>
     </property>
     <property name="text">
      <string>Static Languages</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Stopword Ratio to determine the language</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="MinStopWords"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>StaticLang</zorder>
  <zorder>RemoveStopwords</zorder>
  <zorder>buttonBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
